core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -146,12 +146,16 @@ public String getLoadBalancerUserFacingFQDN(Long stackId) {
         return loadBalancerOptional;
     }
 
-    public Set<LoadBalancer> createLoadBalancers(Stack stack, DetailedEnvironmentResponse environment) {
+    public Set<LoadBalancer> createLoadBalancers(Stack stack, DetailedEnvironmentResponse environment, boolean loadBalancerDefined) {
```

Comment:
Could we rename `loadBalancerDefined`? I don't think it's very descriptive, because at this point a load balancer isn't defined. Maybe something like `loadBalancerFlagEnabled`?